src/js/components/Meter/stories/Bar.js

```diff
@@ -4,12 +4,14 @@ import { storiesOf } from '@storybook/react';
 import { Grommet, Box, Meter } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-const BarMeter = () => (
-  <Grommet theme={grommet}>
-    <Box align="center" pad="large">
-      <Meter type="bar" background="light-2" values={[{ value: 30 }]} />
-    </Box>
-  </Grommet>
-);
+function BarMeter() {
```

Comment:
why was it changed to a function rather than a const?

Reply:
I've changed them now, didn't notice that the default was lambda's in this project